File: JavaScript/Brick.js
class Brick extends Sprite{
    constructor(x, y, w, h, model){
        super(x, y, w, h, model);
        this.model = model;
        this.isBrick = true;
    }

    isBrick(){
        return true;
    }

    update(){
        //nothing needs to be updated
    }

    draw(ctx){
        ctx.strokeStyle="##18EFF";
        ctx.rect(this.x - this.model.screenPos, this.y, this.w, this.h);
        ctx.stroke();
    }

}

File: Python/Brick.py
import pygame
from Sprite import *

class Brick(Sprite):

    def __init__(self, model, x, y, w, h):
        super(Brick, self).__init__(x,y,w,h)
        self.model = model
        isBrick = True

    def update(self):
        pass

    def draw(self, screen):
        pygame.draw.rect(screen, (255,255,255), [self.x-self.model.screenPos, self.y, self.w, self.h], 3)

        


